manifestservice/manifests/__init__.py

```diff
@@ -354,7 +496,10 @@ def _list_files_in_bucket(bucket_name, folder):
         "cohorts": [
             # For files in the cohorts/ folder
             { "filename": <filename>, "last_modified": <timestamp> }, ...
-        ]
+        ],
+        "metadata": [
+            { "external_oidc_idp": "qdr-keycloak", "file_retriever": "QDR", "study_id": "doi:10.5064/F6N2GOC9" }, ...
```

Comment:
```suggestion
            # For files in the exported-metadata/ folder
            { "filename": <filename>, "last_modified": <timestamp> }, ...
```
I believe this is what it suppose to return?